docs/components/board/pi5.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Configure a Raspberry Pi 5 board"
+linkTitle: "pi5"
+weight: 20
+type: "docs"
+description: "Configure a Raspberry Pi 5 board."
+images: ["/icons/components/board.svg"]
+tags: ["board", "components"]
+# SMEs: Bucket, Alan
+---
+
+{{% alert title="REQUIREMENTS" color="note" %}}
+
+Follow the [setup guide](/get-started/installation/prepare/rpi-setup/) to prepare your Pi for running `viam-server` before configuring a `pi5` board.
+
+{{% /alert %}}
+
+Configure a `pi5` board to integrate a [Raspberry Pi 5](https://www.raspberrypi.com/products/raspberry-pi-5/) into your robot.
+
+To configure a Raspberry Pi 4 or earlier, see [Configure a Raspberry Pi 4, 3, or Zero 2 board](/components/board/pi/).
+
+### Enable hardware PWM
+
+_(Optional)_ If you want to use hardware PWM on {{< glossary_tooltip term_id="pin-number" text="pins" >}} 12 and 35, edit <file>/boot/config.txt</file> on your Pi, adding the following line:
+
+```sh {class="line-numbers linkable-line-numbers"}
+dtoverlay=pwm-2chan
+```
+
+Then reboot the Pi for the change to take effect.
+
+### Configuration
+
+{{< tabs name="Configure a pi5 Board" >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and click **Create component**.
+Select the `board` type, then select the `pi5` model.
+Enter a name for your board and click **Create**.
+
+![An example board configuration in the app builder UI. The name (local), type (board) and model (pi5) are shown. No other attributes are configured.](/tutorials/scuttlebot/board-empty-json.png)
```

Comment:
Description doesn't match image: the image shows a pi (non-5) board. Maybe I expect too much from our users, but it's not clear to me that we even need a screenshot here.

Reply:
Yes, you're quite right--The app UI doesn't seem to have the pi5 in it yet so I'm using this as a placeholder until I can take a proper screenshot. Do you know when it will be available there?

Reply:
:grimacing: That's a great question! It's due out in the stable RDK release this coming Tuesday, but I thought the app did, like, daily deployments or something, and if it's not in there yet, perhaps I need to add something else to get that to work. but I don't know anything about app stuff. Do you know who on the Fleet teams I should talk to to track this down?

Reply:
Thanks for prompting me on this! I've merged https://github.com/viamrobotics/app/pull/3114, which adds fancy configs for all the other boards I've made, and will hopefully have fancy configs for the pi5 this coming Tuesday or Wednesday. In the meantime (or perhaps even after that, since it's not a big deal?), let's leave this as you've written it.